config/locales/en.yml

```diff
@@ -1197,7 +1197,7 @@ in_person_proofing.body.barcode.questions: Questions?
 in_person_proofing.body.barcode.retail_hours: Retail hours
 in_person_proofing.body.barcode.retail_hours_closed: Closed
 in_person_proofing.body.barcode.return_to_partner_html: You may now %{link_html} to complete any next steps you can access until your identity has been verified.
-in_person_proofing.body.barcode.return_to_partner_link: sign out and return to %{sp_name}
+in_person_proofing.body.barcode.return_to_partner_link: Return to %{sp_name}
```

Comment:
`in_person_proofing.body.barcode.return_to_partner_link` is being used in two places. 

1. app/views/idv/in_person/ready_to_verify/show.html.erb
2. spec/support/features/idv_step_helper.rb

Do you want want to change the text in idv_step_helper?

Reply:
Yes, I believe that helper in `idv_step_helper` used to build the link text so it can be tested to ensure that content is the same

Reply:
This might be a better question for content.  I would track the views that this happens- and screenshot- and ask. Because this effects views possible outside of IPP- I think you should get the green light from content.